admin.py

```diff
@@ -274,7 +286,11 @@ def compile_resources(
     target_path = output_directory / "resources.py"
     target_path.parent.mkdir(parents=True, exist_ok=True)
     _log(f"compile_resources target_path: {target_path}", context=context)
-    subprocess.run(shlex.split(f"pyrcc5 -o {target_path} {resources_path}"))
+    subprocess.run(
+        shlex.split(
+            f"pyrcc5 -o {target_path.as_posix()} " f"{resources_path.as_posix()}"
```

Comment:
The original path type is intended to work on all platflorms, what is the benefit of using a PosixPath.

Reply:
Thanks Samweli. The paths use the default backslash in Windows and this does not work well with shlex.split() hence the need to force it to use posix path.

Reply:
Okay, then we can check the platform os and then use the right path type, similar to what you have done when getting application data path for the QGIS profiles folder.